programs/bpf_loader/src/deprecated.rs

```diff
@@ -2,5 +2,6 @@ solana_sdk::declare_builtin!(
     solana_sdk::bpf_loader_deprecated::ID,
     solana_bpf_loader_deprecated_program,
     solana_bpf_loader_program::process_instruction,
+    1140,
```

Comment:
nit: Is there a reason for defining these as strict numbers rather than with respect to `COMPUTE_UNIT_TO_US_RATIO`, e.g. this one used to be `COMPUTE_UNIT_TO_US_RATIO * 38`?

Reply:
Good point, I am not settled on what it should be. The initial thought was it should just be an organic number of CUs, based on dev's understanding/testing of the program. Pulling in another concept, eg `COMPUTE_UNIT_TO_US_RATIO`, in defining its cost may not be worth it. Perhaps add comments for additional info of chosen value?

Reply:
I believe the idea is that each compute unit should take 30us on a dev machine when benchmarked, paging @samkim-crypto on this again since he was looking at this recently :sweat_smile:

Reply:
Yes, for syscalls, I was relying on this [comment](https://github.com/solana-labs/solana/pull/25464#issuecomment-1142598915), which circularly quotes @taozhu-chicago 😅. According to this, the cluster average is 30 CU per 1 us, which translates to 1/30 = 0.033us or ~33ns per CU.

Reply:
Haha all right, sounds like Tao has all the power then!